docker/docker-compose.yml

```diff
@@ -28,7 +25,7 @@ x-php: &php
     PAGER: more
     HM_ENV_ARCHITECTURE: 'local-server'
     HM_DEPLOYMENT_REVISION: 'dev'
-    ELASTICSEARCH_HOST: elasticsearch
+    ELASTICSEARCH_HOST: elasticsearch.${COMPOSE_PROJECT_NAME:-default}.altis.dev
```

Comment:
Shouldn't hte ES instance still be addressable on the elasticseearch (only) hostname?

Reply:
If we do that, we run into the same problem as before where more than 1 elasticsearch container IP will be assigned to that hostname because they all exist on the same network (proxy). If the elasticsearch service doesn't need to be addressable outside of it's environment, then we can remove the Traefik labels and address it via the `elasticsearch`. Is that clear as mud? 😛

Reply:
Ohh gotcha, forgot we were putting it on the proxy network. I think we _do_ want it externally addressable, so that's 👍 then.